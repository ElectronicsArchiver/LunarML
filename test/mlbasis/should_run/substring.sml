val (a, b) = Substring.splitl (fn c => c <> #"X") (Substring.full "aaaXbbbbXccc")
val () = print (Substring.string a ^ "|" ^ Substring.string b ^ "\n")
val (a, b) = Substring.splitr (fn c => c <> #"X") (Substring.full "aaaXbbbbXccc")
val () = print (Substring.string a ^ "|" ^ Substring.string b ^ "\n")
val a = Substring.triml 3 (Substring.full "foobar")
val () = print (Substring.string a ^ "\n");
val a = Substring.trimr 3 (Substring.full "foobar")
val () = print (Substring.string a ^ "\n");
