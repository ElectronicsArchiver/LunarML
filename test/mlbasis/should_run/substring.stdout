aaa|XbbbbXccc
aaaXbbbbX|ccc
bar
foo
